docs/docs-content/enterprise-version/upgrade/upgrade-k8s/airgap.md

```diff
@@ -0,0 +1,432 @@
+      ```
+
+    </TabItem>
+
+    </Tabs>
+
+3.  Prepare and export the environment variables required by the Palette airgap binary. Select one of the following tabs
+    for instructions on how to prepare these variables for Harbor and AWS ECR.
+
+    <Tabs>
+
+    <TabItem value="harbor" label="Harbor" default>
+
+    - `OCI_IMAGE_REGISTRY`: The IP address or domain name of the `spectro-images` registry.
+    - `OCI_IMAGE_BASE`: The namespace or repository that hosts the images.
+    - `OCI_PACK_REGISTRY`: The IP address or domain name of the `spectro-packs` registry.
+    - `OCI_PACK_BASE`: The namespace or repository that hosts the packages.
+
+      ```shell
+      export OCI_IMAGE_REGISTRY=<harbor-endpoint>
+      export OCI_IMAGE_BASE=spectro-images
+      export OCI_PACK_REGISTRY=<harbor-endpoint>
+      export OCI_PACK_BASE=spectro-packs
+      ```
+
+    Consider the following example for reference.
+
+    ```shell
+    export OCI_IMAGE_REGISTRY=example.internal.com
+    export OCI_IMAGE_BASE=spectro-images
+    export OCI_PACK_REGISTRY=10.10.100.48
+    export OCI_PACK_BASE=spectro-packs
+    ```
+
+    </TabItem>
+
+    <TabItem value="aws-ecr" label="AWS ECR">
+
+    - `ECR_IMAGE_REGISTRY`: The IP address or domain name of the public `spectro-images` registry.
+    - `ECR_IMAGE_BASE`: The namespace or repository that hosts the images.
+    - `ECR_IMAGE_REGISTRY_REGION`: The AWS region where the ECR registry is located.
+    - `ECR_PACK_REGISTRY`: The IP address or domain name of the public `spectro-packs` registry.
+    - `ECR_PACK_BASE`: The namespace or repository that hosts the packages.
+    - `ECR_PACK_REGISTRY_REGION`: The AWS region where the ECR registry is located.
+
+      ```shell
+      export ECR_IMAGE_REGISTRY=<ecr-endpoint>
+      export ECR_IMAGE_BASE=spectro-images
+      export ECR_IMAGE_REGISTRY_REGION=<ecr-region>
+      export ECR_PACK_REGISTRY=<ecr-endpoint>
+      export ECR_PACK_BASE=spectro-packs
+      export ECR_PACK_REGISTRY_REGION=<ecr-region>
+      ```
+
+    Consider the following example for reference.
+
+    ```shell
+    export ECR_IMAGE_REGISTRY=public.ecr.aws/1234567890
+    export ECR_IMAGE_BASE=spectro-images
+    export ECR_IMAGE_REGISTRY_REGION=us-east-1
+    export ECR_PACK_REGISTRY=123456789.dkr.ecr.us-east-1.amazonaws.com
+    export ECR_PACK_BASE=spectro-packs
+    export ECR_PACK_REGISTRY_REGION=us-east-1
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+4.  Download the airgap setup binary using the credentials our support team provided along with the binary version. Use
+    the following command template to set up a `VERSION` variable you'll use in the following steps.
+
+    ```shell
+    export VERSION=<x.y.z>
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    export VERSION=4.2.7
+    ```
+
+5.  Use the following command template to download the airgap setup binary.
```

Comment:
This makes me a bit nervous. I think we should tell them to use the URL that support provides them and ensure they provide the credentials provided. Just like we told them in the VMware step.

> Use the following command template to download the new Palette airgap installation bin. Enter the username, password, and the Palette airgap installation URL you received from our support team. In the output file name, replace <version> with the Palette version you're downloading.